common/seeder/package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "sublinks-job-seeder",
+  "version": "1.0.0",
+  "scripts": {
+    "seed": "node ./job-seeder.js"
+  },
+  "dependencies": {
+    "sublinks-js-client": "^0.0.3"
```

Comment:
Version 0.0.13 is the latest one. This could be a good opportunity to upgrade the seeder dependency.

Reply:
Updated. Also added Dependabot config to (hopefully) keep it updated